lighthouse-cli/test/cli/run-test.js

```diff
@@ -58,8 +58,21 @@ describe('Parsing --chrome-flags', () => {
     assert.deepStrictEqual([], parseChromeFlags());
   });
 
+  it('keeps --no-flags untouched, #3003', () => {
+    assert.deepStrictEqual(['--no-sandbox'], parseChromeFlags('--no-sandbox'));
+  });
+
+  it('handles numeric values', () => {
+    assert.deepStrictEqual(['--log-level=0'], parseChromeFlags('--log-level=0'));
+  });
+
   it('returns flags correctly for issue 2817', () => {
```

Comment:
while you're in here, mind adding something like `it('quotes flag values with spaces in them'), () => {` or whatever (rather than just "issue 2817") :)